src/pages/index.js

```diff
@@ -1,12 +1,14 @@
 import React, { useEffect, useState } from "react";
 import { Blurhash } from "react-blurhash";
+import { SiMicrosoftbing } from "react-icons/si";
+import { RiGoogleLine } from "react-icons/ri";
```

Comment:
For Google Icon, how about we use from mui icons?

`import GoogleIcon from '@mui/icons-material/Google';`